avatars/avatars.js

```diff
@@ -562,11 +562,12 @@ class Avatar {
     this.eyeballTarget = new THREE.Vector3();
     this.eyeballTargetPlane = new THREE.Plane();
     this.eyeballTargetEnabled = false;
-
+    this.springBoneManager = null;
     if (options.hair) {
-      this.springBoneManager = new VRMSpringBoneImporter().import(object);
-    } else {
-      this.springBoneManager = null;
+      (async () => {
+        this.springBoneManager = await (new VRMSpringBoneImporter()).import(object);
```

Comment:
I don't think this can be async. I believe the existing `three-vrm` fork specifically modifies `VRMSpringBoneImporter` to be sync, because it is unacceptable for a constructor to be async.